sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -780,6 +780,31 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
     )
   }
 
+  test("map_sort function") {
```

Comment:
Could you cover more corner cases like:
1. empty map
2. duplicate keys, see the config `spark.sql.mapKeyDedupPolicy`
3. null keys
4. check the error class `INVALID_ORDERING_TYPE`